tests/appgeo/gdal.pm

```diff
@@ -0,0 +1,25 @@
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install gdal and perform tests
+# Maintainer: Guillaume <guillaume@opensuse.org>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
```

Comment:
I'd use serial console here (it's just faster)

Reply:
Not sure how to do this. And, I need root permission for zypper.

Reply:
serial console is runs under root , maybe you can try it out ?

Reply:
> serial console is runs under root , maybe you can try it out ?

Do you have an example how to do it, please?

Reply:
It does dun under root.